### ISSUE TITLE
[PR commands] help

### ISSUE THREAD

**Issue by yihau** [solana-labs/move#359]

**Comment by yihau** [comment]
I think this one has already implemented.

https://github.com/solana-labs/move/blob/16ed36ab8d0815dd25372107765619078306109f/.github/workflows/pr-commands.yml#L57-L79

**Comment by ksolana** [comment]
!help

**Comment by ksolana** [comment]
doesn't seem to work?

**Comment by yihau** [comment]
this bot only works for a PR atm. I can send a PR to get it work for issues as well if we need

**Comment by ksolana** [comment]
works! it only has one command for now it seems.

https://github.com/solana-labs/move/pull/368#issuecomment-1722815008

**Comment by yihau** [comment]
yeah. unfortunately, the only one command will be removed soon by #353.
I think we can get the bot receive commands for issues when we have some `issue` related commands maybe 🤔

**Comment by ksolana** [comment]
ok let's not worry about it then. ok to close.
